src/main/java/com/github/cameltooling/lsp/internal/codeactions/CamelKafkaConnectorCamelURLRefactor.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cameltooling.lsp.internal.codeactions;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.CodeActionKind;
+import org.eclipse.lsp4j.CodeActionParams;
+import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextDocumentItem;
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4j.WorkspaceEdit;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+
+import com.github.cameltooling.lsp.internal.CamelTextDocumentService;
+import com.github.cameltooling.lsp.internal.instancemodel.CamelURIInstance;
+import com.github.cameltooling.lsp.internal.parser.CamelKafkaUtil;
+import com.github.cameltooling.lsp.internal.parser.ParserFileHelper;
+import com.github.cameltooling.lsp.internal.parser.ParserFileHelperFactory;
+import com.github.cameltooling.lsp.internal.parser.ParserFileHelperUtil;
+
+public class CamelKafkaConnectorCamelURLRefactor {
```

Comment:
Hi @apupier , this class provides a code action, right? What do you think about renaming the class to something like `ConvertUriToPropertiesAction` or similar ?

Reply:
it is an `Action` of type `Refactor`.
I would like to keep the notion of Camel Kafka Connector.
maybe `ConvertCamelKafkaConnectorURLToPropertiesRefactorAction`? but is is starting to be a long class name

Reply:
or we go with the CKC contraction? `ConvertCKCUrlToPropertiesRefactorAction` / `CKCConvertCamelUrlToPropertiesRefactorAction` / `CKCConvertCamelUrlToPropertiesRefactor`

Reply:
I would not use abbreviations tbh. `ConvertCamelKafkaConnectorURLToPropertiesRefactorAction` sounds good to me.

Reply:
It may be long, but it is definitely clearer in the longer form